molgenis-data/src/main/java/org/molgenis/data/meta/MetaUtils.java

```diff
@@ -37,11 +36,8 @@ public static Fetch getEntityTypeFetch()
 	 */
 	public static boolean isSystemPackage(Package aPackage)
 	{
-		//Needs to run as system since you might not have permission to read the sys package
-		//Once we have READ_META permission checks on the packages this runAsSystem can be removed
-		//https://github.com/molgenis/molgenis/issues/7528
-		return runAsSystem(() -> aPackage.getId().equals(PACKAGE_SYSTEM) || (aPackage.getRootPackage() != null
-				&& aPackage.getRootPackage().getId().equals(PACKAGE_SYSTEM)));
+		return aPackage.getId().equals(PACKAGE_SYSTEM) || (aPackage.getRootPackage() != null
```

Comment:
READ_METADATA was introduced in a8180c95fa0cdbfb6d8310be2e1ceab9955ec05b, the check removed in this PR was introduced afterwards in e1ec52d55dc2c3d43497609961d95f1b254bb7fe. Can it safely be removed?